src/types/account.js

```diff
@@ -152,11 +153,15 @@ export type Account = {
   // balance history represented the balance evolution throughout time, used by chart.
   // This is to be refreshed when necessary (typically in a sync)
   // this is a map PER granularity to allow a fast feedback when user switch them
-  balanceHistory?: BalanceHistoryMap
+  balanceHistory?: BalanceHistoryMap,
+
+  // On some blockchain, an account can have resources (gained, delegated, ...)
+  resources?: ResourcesLike
```

Comment:
Account would deserve to be type parametric but it's not an easy problem.

For now, I feel we should call that `tron` because at the moment you don't offer a way to distinguish with something like a `type` so it will not be easy in future if we want to reuse this and be backward compatible for the raw conversion logic.

we can imagine each family could have a field in account. Maybe it's too early to generalize that, but we can later rewrite it with something like `...AccountSpecifics` the same way we did for the libcore types and how we kept things modular per family. (meaning, types/account shouldn't directly "import from tron" but could import from generated).